Examples/Sources/ChatbotExample/Views/MessageBubble.swift

```diff
@@ -0,0 +1,59 @@
+import SwiftCrossUI
+import Foundation
+
+// MARK: - Message Bubble View
+
+struct MessageBubble: View {
+    let message: ChatMessage
+    
+    var body: some View {
+        HStack(spacing: AppSpacing.sm) {
+            if message.isUser {
+                Spacer()
+                
+                VStack(alignment: .trailing, spacing: AppSpacing.xs) {
+                    Text(message.content)
+                        .font(AppFonts.body)
+                        .foregroundColor(.white)
+                        .padding(AppSpacing.md)
+                        .background(AppColors.primary)
+                        .cornerRadius(AppCornerRadius.large)
+                        .frame(maxWidth: 600, alignment: .trailing)
+                    
+                    Text(formatTime(message.timestamp))
+                        .font(AppFonts.caption2)
+                        .foregroundColor(AppColors.textSecondary)
+                }
+                
+                Text("👤")
+                    .font(AppFonts.title2)
+            } else {
+                Text("🤖")
+                    .font(AppFonts.title2)
+                
+                VStack(alignment: .leading, spacing: AppSpacing.xs) {
+                    Text(message.content)
+                        .font(AppFonts.body)
+                        .foregroundColor(AppColors.text)
+                        .padding(AppSpacing.md)
+                        .background(AppColors.surface)
+                        .cornerRadius(AppCornerRadius.large)
+                        .frame(maxWidth: 600, alignment: .leading)
+                    
+                    Text(formatTime(message.timestamp))
+                        .font(AppFonts.caption2)
+                        .foregroundColor(AppColors.textSecondary)
+                }
```

Comment:
This is repeated from the user version above. Extract the logic for converting a message to a view into a seperate function like so:

```swift
func renderMessage(_ message: ChatMessage) -> some View {
    VStack(...) { ... }
}
```